plugins/org.eclipse.fordiac.ide.debug/src/org/eclipse/fordiac/ide/debug/fb/FBLaunchConfigurationAttributes.java

```diff
@@ -29,6 +31,14 @@ public interface FBLaunchConfigurationAttributes extends LaunchConfigurationAttr
 	String REPEAT_EVENT = "org.eclipse.fordiac.ide.debug.repeatEvent"; //$NON-NLS-1$
 	String KEEP_RUNNING_WHEN_IDLE = "org.eclipse.fordiac.ide.debug.keepRunningWhenIdle"; //$NON-NLS-1$
 
+	String CLOCK_MODE = "org.eclipse.fordiac.ide.debug.clockMode"; //$NON-NLS-1$
+
+	String SYSTEM = "system"; //$NON-NLS-1$
+	String INCREMENT = "increment"; //$NON-NLS-1$
+	String MANUAL = "manual"; //$NON-NLS-1$
```

Comment:
I would suggest defining an enum for these values and using `toString()`/`valueOf()` to convert to/from a string representation for persistence.

Reply:
good idea